chalicelib/checks/wrangler_checks.py

```diff
@@ -472,3 +472,172 @@ def patch_file_size(connection, **kwargs):
     action.status = 'DONE'
     action.output = action_logs
     return action
+
+
+@check_function()
+def biosource_cell_line_value(connection, **kwargs):
+    '''
+    checks cell line biosources to make sure they have an associated ontology term
+    '''
+    check = init_check_res(connection, 'biosource_cell_line_value')
+
+    cell_line_types = ["primary cell", "primary cell line", "immortalized cell line",
+                       "in vitro differentiated cells", "induced pluripotent stem cell line",
+                       "stem cell", "stem cell derived cell line"]
+    biosources = ff_utils.search_metadata('search/?type=Biosource', ff_env=connection.ff_env, page_limit=200)
```

Comment:
Could this be `search/?type=Biosource&frame=object`? Would be a bit more performant and it doesn't look like the check uses any embedded fields